Source/Scene/Scene.js

```diff
@@ -1310,6 +1456,15 @@ define([
 
             scene._globeDepth.executeCopyDepth(context, passState);
 
+            if (scene.debugShowGlobeDepth) {
```

Comment:
Instead of having this explicit copy and managing the framebuffer/textures for debugging yourself, why not have an additional array of `GlobeDepth` (like `_globeDepth`) for debugging?  When `debugShowGlobeDepth` is true, this function would use the frustum to index into the array and use that instead of `_globeDepth`.  This would avoid the copy and remove a lot of code.

@bagnell any reason this wouldn't work?